spec/02-schema_spec.lua

```diff
@@ -6,70 +6,85 @@ local mock_fn_three = 'local x = 1 return function() x = x + 1 end'
 local mock_fn_invalid = 'print('
 local mock_fn_invalid_return = 'return "hello-world"'
 
-for _, plugin_name in ipairs({ "pre-function", "post-function" }) do
 
-  describe(plugin_name .. " schema", function()
+for _, method in ipairs({ "phase+functions", "phase=functions"}) do
+  local function get_conf(functions)
+    if method == "phase+functions" then
+      return { functions = functions }
+    elseif method == "phase=functions" then
+      return { access = functions }
+    end
+  end
+
+  local function get_functions_from_error(err)
+    if method == "phase+functions" then
+      return err.config.functions
+    elseif method == "phase=functions" then
+      return err.config.access
+    end
+  end
+
+for _, plugin_name in ipairs({ "pre-function", "post-function" }) do
```

Comment:
there is an indent missing here

Reply:
I missed it on purpose to keep the history on the specs, since nothing changed in there